blog/2015/04/SCF_minutes.html

```diff
@@ -0,0 +1,52 @@
+---
+layout: blog
+root: ../../..
+author: Karin Lagesebn
```

Comment:
On 07.04.2015 15:57, Raniere Silva wrote:
> In blog/2015/04/SCF_minutes.html
> <https://github.com/swcarpentry/site/pull/920#discussion_r27879491>:
>
>> @@ -0,0 +1,52 @@
>> +---
>> +layout: blog
>> +root: ../../..
>> +author: Karin Lagesebn
>
> @karinlag <https://github.com/karinlag> Could you check your last name?
>  From others post is "Lagesen" but here has a extra |b|. Thanks to
> @arlissc <https://github.com/arlissc> to find this.

While I remember, is there a way to add a title? I am not posting this 
as me, per se, but rather as the SCF secretary.

Karin